cpp/src/aztec/stdlib/encryption/schnorr/schnorr.test.cpp

```diff
@@ -179,11 +179,13 @@ TEST(stdlib_schnorr, test_scalar_mul_low_high)
 /**
  * @brief Death test for cases exluded by `ASSERT` in variable_base_mul.KH
  */
+#if !defined(__wasm__)
 TEST(stdlib_schnorrDeathTest, test_scalar_mul_low_high)
 {
     // without the assert causing the death here, the test passes (i.e., verification fails).
     ASSERT_DEATH(run_scalar_mul_test(0, false), "");
```

Comment:
This is due to an `ASSERT` that gets triggered over at [stdlib/encryption/schnorr/schnorr.cpp#L158](https://github.com/AztecProtocol/barretenberg/blob/7281cf6201518cd0473b9e5372b174dfb470145a/cpp/src/aztec/stdlib/encryption/schnorr/schnorr.cpp#L158). Not sure what we usually do in circuits, but maybe we could `throw_or_abort` instead?

Reply:
Looked into the function, it is justified that the `ASSERT` that was triggered before should never happen. 
It's been removed so that the test can now be run without the DeathTest.